app/models/concerns/conclusion_reviews/gal_pdf.rb

```diff
@@ -597,14 +622,15 @@ def collapsed_control_objectives_row_data
       row_data
     end
 
-    def control_objectives_row_data brief
+    def control_objectives_row_data brief, scope_detail
       count         = 0
       row_data      = []
       image_options = { vposition: :top, border_widths: [1, 0, 1, 0] }
 
       review.grouped_control_objective_items.each do |process_control, cois|
         cois.sort.each do |coi|
           text  = coi.control_objective_text
+          text  = text.lines.first.upcase if scope_detail
```

Comment:
```suggestion
          text = if scope_detail
                   text.lines.first.upcase 
                 else
                   coi.control_objective_text
                 end
```

Queda raro de la otra forma, no es muy evidente lo que pasa.